sunpy/map/tests/test_mapbase.py

```diff
@@ -206,6 +206,11 @@ def test_cmap(generic_map):
     assert generic_map.cmap == plt.get_cmap('gray')
 
 
+def test_norm(generic_map):
+    from matplotlib import colors
+    assert isinstance(generic_map.norm, colors.Normalize)
+
```

Comment:
Do we want a test to ensure you can create a map with norm and cmap?

Reply:
Sorry I didn't understood

Reply:
We added two new keywords to the init but unless I missed it (did the review late at night), I don't see a test to ensure that they work?

Reply:
Oh sorry, I missed adding the test! Will add it soon!

Reply:
See my comment below w.r.t. tests